openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -1429,11 +1429,15 @@ def get_parameter(self, parameter_attrs):
 
         Returns
         -------
-        list of ParameterType objects
+        params : list of ParameterType objects
             A list of matching ParameterType objects
         """
-        # TODO: This is a necessary API point for Lee-Ping's ForceBalance
-        raise NotImplementedError()
+        params = list()
+        for attr, value in parameter_attrs.items():
+            for param in self._parameters:
+                if getattr(param, attr) == value:
```

Comment:
Would this double-add parameters if a parameter matched on two queried values? It would be good to ensure we only return unique parameters once.